### PR TITLE
Add pagination to getOwned endpoints

### PR DESCRIPTION
**Pull request by kien-ngo**
## Changes

Add pagination to getOwned endpoints

## How this PR will be tested

- [ ] Open the dashboard and click X. Result: A modal should appear.
- [x] Call the /foo/bar API. Result: Returns 200 with "baz" in the response body.

## Output

(Example: Screenshot/GIF for UI changes, cURL output for API changes)

### CODE REVIEW

**Comment by kien-ngo** [comment]
cc.  @joaquim-verges

File: server/api/contract/extensions/erc1155/read/getOwned.ts [hunk 1]
```diff
@@ -16,6 +16,18 @@ const querystringSchema = Type.Object({
     description: "Address of the wallet to get NFTs for",
     examples: ["0x1946267d81Fb8aDeeEa28e6B98bcD446c8248473"],
   }),
+  start: Type.Optional(
+    Type.Number({
+      description: "The start token id for paginated results. Defaults to 0.",
```
Comment (joaquim-verges) [inline server/api/contract/extensions/erc1155/read/getOwned.ts:21]: Not necessarily the token id, more like the starting index

Comment (kien-ngo) [reply server/api/contract/extensions/erc1155/read/getOwned.ts:21]: Oh, I copied the whole text from the `getAll` endpoint. Let me update

**Comment by kien-ngo** [comment]
cc. @joaquim-verges

**Pull request closed without merging**